Library/Homebrew/cmd/help.rb

```diff
@@ -32,14 +32,34 @@
 # NOTE The reason the string is at the top is so 25 lines is easy to measure!
 
 module Homebrew
-  def help
-    puts HOMEBREW_HELP
-  end
+  def help(cmd = nil, empty_argv = false)
```

Comment:
Very likely. I presume the call site will always be either `Homebrew.help cmd, empty_argv` or just `Homebrew.help`, i.e. there will never be a literal second argument.